styles/twitch/catppuccin.user.css

```diff
@@ -680,6 +689,45 @@
     .modal__content {
       color: @text;
     }
+
+    /* Leaderboard highlighted username */
+    .bits-leaderboard-expanded-top-three-entry__marquee-username .gRUYWI {
+      color: @base !important;
+    }
+
+    /* Privacy center */
+
+    .evszEp {
+      background: @base !important;
+    }
+
+    .jTEslw {
+      background: @base !important;
+    }
+
+    .evszEp,
+    .jTEslw {
+      svg path {
+        fill: @accent-color !important;
+        &[fill="#fff"],
+        &[fill="#FFF"] {
+          fill: none !important;
+        }
+      }
+    }
```

Comment:
Same as above, let's avoid autogenerated classnames as these will break frequently.

Reply:
It seems that these parts cannot be themed in any other way.

Reply:
Can you show me the HTML for these? There aren't any aria attributes, IDs, inline styles, unique element structures we can target?

Reply:
Nevermind, I figured it out.